### PR TITLE
Added write_raw to gpib_server.py

### PR DESCRIPTION
**Pull request by dahlend**
I had a typo in my original pull request.

### CODE REVIEW

File: gpib_server.py [hunk 1]
```diff
@@ -165,6 +165,11 @@ def write(self, c, data):
         """Write a string to the GPIB bus."""
         self.getDevice(c).write(data)
 
+    @setting(8, data='s', returns='')
```
Comment (maffoo) [inline gpib_server.py]: In the recent (scala) versions of the manager there is a new type `'y'` for binary data. The intent is that `'s'` is for text strings in utf8 encoding, while `'y'` can contain arbitrary bytes. Note that the `read_raw` setting below returns `'y'`; we should use that type here as well.

**Comment by maffoo** [comment]
Since this is adding a setting, please bump the server version in the node config block at the top to `1.5.0-no-refresh`.

**Comment by maffoo** [comment]
Please see comments, then I'm happy to merge. Note that you can just push additional commits to your branch and this PR will get updated; no need to open a new PR.

**Comment by dahlend** [comment]
OK, changes made.

**Comment by maffoo** [comment]
Thanks, @dahlend!

**Pull request merged**